Helper/Data.php

```diff
@@ -125,29 +134,29 @@ public function getRegion($storeId)
         }
         return $region;
     }
-
     
     // General Config
-    public function isSearchActive($storeId = null)
```

Comment:
Removing search - isMerchActive always returns true, isSearchActive always returns false; existing code left commented out for legacy search users